src/js/client/payment.js

```diff
@@ -27,13 +27,13 @@ module.exports = function () {
             method: 'POST',
             data: data
           })
-            .success((data) => {
-              $('.payment-form').remove();
-              $('.payment-form-description').text(data.message);
+            .done((data) => {
+              $output.text(data.message);
             })
             .fail((jqXHR) => {
               let errormsg = ((jqXHR.responseJSON) && (jqXHR.responseJSON.error)) ? jqXHR.responseJSON.error : 'Something went wrong. Please try again.';
               $output.text(errormsg);
```

Comment:
![screen shot 2017-12-20 at 00 33 49](https://user-images.githubusercontent.com/14129055/34185536-ceec6642-e51d-11e7-90f4-dcef773247e1.png)

In the event of an error, we may wish to have some additional text asking the user to try again.  In the simplest form this could just be saying "Please try making the payment again" — we could also include a link to our contact email.